modules/_node-scripts/esbuild/bundleJavaScriptExports.mjs

```diff
@@ -0,0 +1,41 @@
+import getExactAliasPlugin from './getExactAliasPlugin.mjs';
+import getFlatName from '../util/getFlatName.mjs';
+import getEntryPoint from './getEntryPoint.mjs';
+import getExternals from './getExternals.mjs';
+import getImportBridgesPlugin from './getImportBridgesPlugin.mjs';
+import runEsbuild from './runEsbuild.mjs';
+import writeExportBridge from './writeExportBridge.mjs';
+
+export default async function bundleJavaScriptExports(
+	globalImports, globalSymbols, projectExports
+) {
+	if (!projectExports.length) {
+		return;
+	}
+
+	await Promise.all(
+		projectExports
+			.filter(moduleName => !moduleName.endsWith('.css'))
```

Comment:
What about `.scss` files? Do we need to worry about those?

Reply:
Nope, because for now we only use allow CSS or JS exports, since they are only used for npm packages (not internal code).

In fact, if you exported a .scss file esbuild would complain because it doesn't know what to do with it :shrimp: .